checker/src/main/java/org/checkerframework/checker/calledmethods/CalledMethodsAnnotatedTypeFactory.java

```diff
@@ -381,37 +380,38 @@ public Void visitExecutable(AnnotatedTypeMirror.AnnotatedExecutableType t, Void
   }
 
   @Override
-  public List<AnnotationMirror> getPostconditionAnnotation(
-      VariableElement elt, AnnotatedTypeMirror fieldAnnos, List<AnnotationMirror> preconds) {
-    AnnotationMirror cmAnno =
-        AnnotationUtils.getAnnotationByName(
-            fieldAnnos.getAnnotations(),
-            "org.checkerframework.checker.calledmethods.qual.CalledMethods");
-    if (cmAnno != null) {
+  protected @Nullable AnnotationMirror requiresOrEnsuresQualifierAnnoForExpression(
```

Comment:
Why is the `ForExpression` part of this name necessary?  There isn't any other `requiresOrEnsuresQualifierAnno` method that needs to be distinguished from this one.